src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -1061,7 +1061,7 @@ def test_replace_with_named_context():
     node1 = Literal('1', INTEGER_TYPE)
```

Comment:
Please could you fix the two useless-object-inheritance warnings in this file.

Reply:
Done